egs/heroico/s5/local/heroico_answers_make_lists.pl

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/perl -w
```

Comment:
We prefer to use /usr/bin/env perl  shebang, i.e.
```
#!/usr/bin/env perl
```

env typically allows only one parameter, so you cannot supply -w switch, but you are using `use warnings` anyway, so it should not be necessry.